tests/cupyx_tests/scipy_tests/special_tests/test_bessel.py

```diff
@@ -57,11 +69,11 @@ def test_yn(self, xp, scp, dtype, order_dtype):
 class TestFusionSpecial(unittest.TestCase):
 
     @testing.for_dtypes(['e', 'f', 'd'])
-    @testing.numpy_cupy_allclose(atol=1e-5, scipy_name='scp')
+    @testing.numpy_cupy_allclose(rtol=1e-5, scipy_name='scp')
     def check_unary(self, name, xp, scp, dtype):
         import scipy.special  # NOQA
 
-        a = testing.shaped_arange((2, 3), xp, dtype)
+        a = xp.linspace(-10, 10, 50, dtype=dtype).reshape((2, -1))
```

Comment:
Sorry, but the CI failure seems related to this change. I will make a fix soon in another PR!

Reply:
#7081